tasks/rulegen/template.txt

```diff
@@ -52,6 +52,26 @@ declare_oxc_lint!(
     /// ```{{language}}
     /// FIXME: Tests will fail if examples are missing or syntactically incorrect.
     /// ```
+{{#if rule_config}}
+    ///
+    /// ### Options
+    ///
+    /// #### optionName
+    ///
+    /// `{ type: FIXME, default: FIXME }`
+    ///
+    /// FIXME: Explanation of what this option does.
+    ///
+    /// Example:
+    /// ```json
+    /// {
+    ///   "{{mod_name}}/{{kebab_rule_name}}": [
```

Comment:
I think this won't work correctly because `mod_name` is going to return with underscores here, but haven't been able to test a non-eslint rule to generate yet